### ISSUE TITLE
Live-input

### ISSUE THREAD

**Issue by Hagonata** [innat/VideoMAE#3]
Is it can handle live video input? With live predictions like
And also can I use weights which I train with VideoMAE from Hugging-Face pipeline? They have this list of files:
![image](https://github.com/innat/VideoMAE/assets/95108691/c1f27ab3-8f0c-4727-8c44-236f47f3dc55)

**Comment by innat** [comment]
> Is it can handle live video input? With live predictions like

Yes, I think it is possible to apply video-mae on live-video. Have you tried it? If you face any issue with this codebases, let me know. This sounds interesting, I might add it later. 

> And also can I use weights which I train with VideoMAE from Hugging-Face pipeline? They have this list of files:

I'm not sure, as the underline framework is different.
